sql-schema/183.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE users_prefs DROP PRIMARY KEY;
+ALTER TABLE users_prefs DROP INDEX pref;
+INSERT INTO `users_prefs` (`user_id`, `pref`, `value`) SELECT `user_id`, 'dashboard', `dashboard` FROM `users` WHERE `dashboard` != '0';
+INSERT INTO `users_prefs` (`user_id`, `pref`, `value`) SELECT `user_id`, 'twofactor', `twofactor` FROM `users` WHERE `twofactor` != '0';
+ALTER TABLE `users` CHANGE `updated_at` `updated_at` TIMESTAMP NOT NULL DEFAULT '1970-01-01 00:00:01';
```

Comment:
Nothing's ever simple. So this still doesn't work when your TZ is not UTC :(

Seems `DEFAULT CURRENT_TIMESTAMP` works.